opengrok-indexer/pom.xml

```diff
@@ -233,6 +233,11 @@ Portions Copyright (c) 2020-2020, Lubos Kosco <tarzanek@gmail.com>.
             <artifactId>annotations</artifactId>
             <version>20.1.0</version>
         </dependency>
+        <dependency>
+            <groupId>org.eclipse.jgit</groupId>
+            <artifactId>org.eclipse.jgit</artifactId>
+            <version>5.10.0.202012080955-r</version>
```

Comment:
Just a note, 5.11 version was released last week: https://mvnrepository.com/artifact/org.eclipse.jgit/org.eclipse.jgit/5.11.0.202103091610-r

However, if it has some breaking changes, then we can create a separate PR for that.

Reply:
should go with the latest. The changes use official APIs only.